tetris.py

```diff
@@ -245,5 +259,11 @@ def full_line_clear(self):
                     (a, b + 1) for (a, b) in self.landed_blocks if b < y_line
                 ] + [(a, b) for (a, b) in self.landed_blocks if b > y_line]
 
+    def block_color(self):
```

Comment:
If we make this take the shape to use as an argument, we can use the same method for coloring landed blocks, once their shapes are remembered.

```suggestion
    def block_color(self, shape):
```